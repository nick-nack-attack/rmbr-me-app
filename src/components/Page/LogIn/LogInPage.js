import React, { Component } from 'react';

// component
import LogInForm from '../../Form/LogIn/LogInForm';

export default class LogInPage extends Component {

    static defaultProps = {
        location: {},
        history: {
            push: () => {},
        }
    };

    handleLoginSuccess = () => {
        window.location.reload(true);
    };

    render() {
        return (
                <LogInForm
                    onLoginSuccess={(ev) => this.handleLoginSuccess(ev)}
                />
        );
    };
};