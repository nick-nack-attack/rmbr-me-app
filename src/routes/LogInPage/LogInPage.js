import React, { Component } from 'react'
import LogInForm from '../../components/LogInForm/LogInForm'
import { Section } from '../../components/Utils/Utils'

export default class LogInPage extends Component {

    static defaultProps = {
        location: {},
        history: {
            push: () => {},
        }
    };

    handleLoginSuccess = () => {
        window.location.reload(true)

    }

    render() {
        return (
            <Section className='LoginPage'>
                <LogInForm
                    onLoginSuccess={(ev) => this.handleLoginSuccess(ev)}
                />
            </Section>
        )
    }
}