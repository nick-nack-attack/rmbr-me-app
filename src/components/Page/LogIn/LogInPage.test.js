import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LogInPage from "./LogInPage";

describe(`LogInPage component`, () => {

    it('renders LogInPage by default', () => {
        const wrapper = shallow(<LogInPage/>)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

})