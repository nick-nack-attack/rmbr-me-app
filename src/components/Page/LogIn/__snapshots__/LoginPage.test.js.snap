// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogInPage component renders LogInPage by default 1`] = `
<LogInForm
  onLoginSuccess={[Function]}
/>
`;
